ci/playbooks/meta_content_provider/meta_content_provider.yml

```diff
@@ -47,7 +47,7 @@
     - name: Build openstack services container when gating repo exists
       when:
         - "'os-net-config' not in zuul_change_list"
-        - _gating_repo.stat.exists
+        - _gating_repo.stat.exists or force_container_build | default('false') | bool
```

Comment:
would be enough if you will do: `or force_container_build | default(false)`.

Reply:
Yes, it will work! thank you @danpawlik !

Reply:
I know that it will work, but why it can not be properly xD